packages/ui-core/package.json

```diff
@@ -30,6 +30,7 @@
     "@keen.io/colors": "^0.1.0",
     "@keen.io/icons": "^0.1.0",
     "color": "^3.1.2",
+    "framer": "^1.1.7",
```

Comment:
As we discussed - we'll try to replace `framer` with `framer-motion` to avoid `immutable` in dependencies.